package.json

```diff
@@ -12,7 +12,10 @@
     "prettier-check": "prettier --check .",
     "prettier-fix": "prettier --write .",
     "test": "vitest",
-    "test:coverage": "vitest run --coverage"
+    "test:coverage": "vitest run --coverage",
+    "typecheck": "tsc",
+    "format": "prettier --write .",
+    "format:check": "prettier --check ."
```

Comment:
Why not use the `prettier-fix` or `prettier-check` that we have built into our scripts? 

You can rename them to `prettier:fix` and `prettier:check`, or simply delete the original built-in commands and update the PULL_REQUEST_TEMPLATE in the checklist. 

Currently, it's manageable because the project is still under control.

Reply:
You can rename them to `prettier:fix` and `prettier:check`, or simply delete the original built-in commands and update the PULL_REQUEST_TEMPLATE in the checklist. 

Currently, it's manageable because the project is still under control.